### ISSUE TITLE
[FEATURE REQUEST] Suggestion for Clock desklet

### ISSUE THREAD

**Issue by Pance21** [linuxmint/cinnamon#11092]
A couple of days ago I upgraded to Mint 21, with Cinnamon 5.4.10. I like using the Clock desklet, and it took me a while to figure out how to customize the date format. It would be very helpful if you added sample code, or define all the option. Personally I use this: %l:%M %p%n%a %e %b
The result is: hh:mm AM/PM; new line Day of week, day of month, month.

Thanks

**Comment by Pance21** [comment]
One more thing... I don't see a way to lock the clock desklet on my desktop. It is too easy to move it by mistake, whereas locking it will prevent any accidental moves. Thanks

**Comment by namanlp** [comment]
Hi, 

Why don't you try Conky instead of clock desklet?

I mean, you can use clock desklet, but also try conky. I know it might not solve your issue, but its just a suggestion.

There pretty nice conky clocks available on internet for sure.

**Comment by mfreeman72** [comment]
I totally agree with the decklet locking feature! I've asked for it a while back in another feature request pertaining to a desklet bug, but nothing ever came of it. It would certainly solve a lot of problems.

**Comment by DirkHaar** [comment]
Funny. When I tried to test that, my desklet clock (clock@cinnamon.org) can't be moved at all... I thought it must be locked (no related setting), but no key/mouse combination works for me.

**Comment by rcalixte** [comment]
* There's an option in the calendar applet that could probably be duplicated in the clock desklet to display the date format syntax options that are supported.

* There's also an option in the Desklets module to toggle locking all desklets in place

![2024-01-13_21-55-29](https://github.com/linuxmint/cinnamon/assets/10281587/743d5e1b-b0cd-451d-a653-40d38dddb142)
